### PR TITLE
Routing stuff

### PR DESCRIPTION
**Pull request by rcooper3307**
Routing overhaul!

### CODE REVIEW

File: src/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+import {Link} from "react-router-dom";
+import './styles/main.scss';
+import Form from './Form.jsx';
+import ExampleComponent from './wintern-bios/ExampleComponent.jsx';
+import RomasComponent from './wintern-bios/RomasComponent.jsx';
+import LarasComponent from './wintern-bios/LarasComponent.jsx';
+
+
+const Home = ({view}) => {
+    console.log('I AM THE VIEW',view);
```
Comment (katdiese) [inline src/Home.jsx:12]: lets clean this up :)

File: src/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    }
+
+    const shouldShowBio = view.length > 0;
+    console.log('shouldShowBio', shouldShowBio);
+    
+
+    return (
+        <>
+    
+
```
Comment (katdiese) [inline src/Home.jsx:35]: lets remove the unnecessary white space here

File: src/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            <a href="#" onClick={() => toggleButton("PATCH",'/bitlinks/{bitlink}')}><span className="Patch">Update</span> a Bitlink</a>
+            <a href="#" onClick={() => toggleButton("PATCH",'/user')}><span className="Patch">Update</span> a User</a>
+            <a href="#" onClick={() => toggleButton("PATCH",'/groups/{group_guid}')}><span className="Patch">Update</span> a Group</a>
+            <a href="#" onClick={() => toggleButton("DELETE",'/groups/{group_guid}')}><span className="Delete">Delete</span> a Group</a>
+            <Link to="/Andrea">Andrea</Link>
+            <Link to="/Tajra">Tajra</Link>
+            <Link to="/Roma">Roma</Link>
+            <Link to="/Lara">Lara</Link>
+            <Link to="/Dana">Dana</Link>
+
```
Comment (katdiese) [inline src/Home.jsx:57]: remove unnecessary whitespace

File: src/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+                  We currently provide our documentation in the form of an OpenAPI 2.0 document. We do not support any code-generation at this time but feel free to use the specification if you would like.
+                </p> 
+                <ExampleComponent myProp={'myProp'} color={'blue'} />
+                <ExampleComponent myProp={'myProp'} />
+                <ExampleComponent myProp={'myProp'} />
+                <ExampleComponent myProp={'myProp'} />
+              </div>
+            }
+          </div>
+          { showForm && !shouldShowBio && <Form endpoint={endpoint} method={method} /> }
```
Comment (katdiese) [inline src/Home.jsx:86]: lets puts `showForm` & `!shouldShowBio` in parenthesis for readability

```
{ (showForm && !shouldShowBio) && ....
```

File: src/Home.jsx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+          { shouldShowBio && <ExampleComponent bioToShow={view} /> }
+          </div>
+        </div>
+        <img className= "logo" alt="bitly logo"src="https://cdn.iconscout.com/icon/free/png-256/bitly-1-432498.png"/>
+        </>
+
+
```
Comment (katdiese) [inline src/Home.jsx:93]: cleanup whitespace

File: src/index.jsx [hunk 6]
```diff
@@ -1,86 +1,64 @@
...
-    const [method, setMethod] = useState('');
-    const [endpoint, setEndpoint] = useState('');
-    const [showForm, setShowForm] = useState(false);
-    const [showIntro, setShowIntro] = useState(true);
-    const [showBio, setShowBio] = useState('');
+    // const [method, setMethod] = useState('');
+    // const [endpoint, setEndpoint] = useState('');
+    // const [showForm, setShowForm] = useState(false);
+    // const [showIntro, setShowIntro] = useState(true);
+    // const [showBio, setShowBio] = useState('');
```
Comment (katdiese) [inline src/index.jsx:22]: remove commented code

File: src/index.jsx [hunk 7]
```diff
@@ -1,86 +1,64 @@
...
+    //   setMethod(method);
+    //   setEndpoint(endpoint);
+    //   setShowForm(true);
+    //   setShowIntro(false);
+    // }
 
-    const shouldShowBio = showBio.length > 0;
-    console.log('showBio', showBio);
+    // const shouldShowBio = showBio.length > 0;
+    // console.log('showBio', showBio);
```
Comment (katdiese) [inline src/index.jsx:32]: remove commented code

File: src/Home.jsx [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+    const toggleButton = (method, endpoint) => {
+      setMethod(method);
+      setEndpoint(endpoint);
+      setShowForm(true);
+      setShowIntro(false);
+    }
+
+    const shouldShowBio = view.length > 0;
+    console.log('shouldShowBio', shouldShowBio);
```
Comment (katdiese) [inline src/Home.jsx:26]: plz remove log

**Review by Andrea-Cajamarca — COMMENTED** [review]
`import RomasComponent from './wintern-bios/RomasComponent.jsx';
import LarasComponent from './wintern-bios/LarasComponent.jsx';`

in Home.jsx, there's still some components that may not be needed anymore

File: src/Home.jsx [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+import RomasComponent from './wintern-bios/RomasComponent.jsx';
+import LarasComponent from './wintern-bios/LarasComponent.jsx';
+
+const Home = ({view}) => {
+    const { useState } = React;
+    const [method, setMethod] = useState('');
+    const [endpoint, setEndpoint] = useState('');
+    const [showForm, setShowForm] = useState(false);
+    const [showIntro, setShowIntro] = useState(true);
+    const [showBio, setShowBio] = useState(view);
```
Comment (katdiese) [inline src/Home.jsx:16]: remove line 16

File: src/Home.jsx [hunk 10]
```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Link} from "react-router-dom";
+import './styles/main.scss';
+import Form from './Form.jsx';
+import ExampleComponent from './wintern-bios/ExampleComponent.jsx';
+import RomasComponent from './wintern-bios/RomasComponent.jsx';
+import LarasComponent from './wintern-bios/LarasComponent.jsx';
```
Comment (katdiese) [inline src/Home.jsx:8]: remove roma & lara components

File: src/index.jsx [hunk 11]
```diff
@@ -1,86 +1,42 @@
...
+import{
+     HashRouter as Router,
+     Switch,
+     Route,
+     Link
+} from "react-router-dom";
 import './styles/main.scss';
 import Form from './Form.jsx';
 import ExampleComponent from './wintern-bios/ExampleComponent.jsx';
+import RomasComponent from './wintern-bios/RomasComponent.jsx';
```
Comment (katdiese) [inline src/index.jsx:12]: don't need romascomponent or examplecomponent

File: src/index.jsx [hunk 12]
```diff
@@ -1,86 +1,42 @@
...
+      <Router>
+        <Switch>
+          <Route path="/Lara">
+            <Home view="Lara"/>
+          </Route>
+          <Route path="/Andrea">
+            <Home view="Andrea"/>
+          </Route>
+          <Route path="/Dana">
+            <Home view="Dana"/>
```
Comment (katdiese) [inline src/index.jsx:27]: change view to bioToShow

File: src/Home.jsx [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+
+const Home = ({view}) => {
```
Comment (katdiese) [inline src/Home.jsx:10]: change view to bioToShow

File: src/Home.jsx [hunk 14]
```diff
@@ -0,0 +1,87 @@
...
+                </p> 
+                <ExampleComponent myProp={'myProp'} color={'blue'} />
+                <ExampleComponent myProp={'myProp'} />
+                <ExampleComponent myProp={'myProp'} />
+                <ExampleComponent myProp={'myProp'} />
+              </div>
+            }
+          </div>
+          { (showForm && !shouldShowBio) && <Form endpoint={endpoint} method={method} /> }
+          { shouldShowBio && <ExampleComponent bioToShow={view} /> }
```
Comment (katdiese) [inline src/Home.jsx:79]: change view to bioToShow

**Review by katdiese — COMMENTED** [review]
requested changes

**Review by katdiese — APPROVED** [review]
👍 LGTM!

**Review by Andrea-Cajamarca — APPROVED** [review]
looks good roma
